### ISSUE TITLE
bug(cy.callFirestore): fixture path being appended with double quotes

### ISSUE THREAD

**Issue by travwritescode** [prescottprue/cypress-firebase#39]
When using the `callFirestore()` function and passing a fixture path as third argument, the path to the fixture is being appended to the full path with double quotes.

So I am trying to set some data in the database from my fixture path `featureFlags/someFeatureFlag.json` which is in my project at this file path: `test/e2e/fixtures/` but I am receiving the error `Error running firebase action: Fixture not found at path: test/e2e/fixtures/"featureFlags/someFeatureFlag.json" or cypress/fixtures/"featureFlags/someFeatureFlag.json"`.

I cannot find where in the code this string is being appended to the file path that is being searched, otherwise I would already have a PR out for this 😝

**Comment by prescottprue** [comment]
Thanks for reporting! That may have something to do with the newer feature for passing data directly instead of just the path.

**Comment by travwritescode** [comment]
@prescottprue The two calls to `JSON.stringify(fixturePath)` in the file `buildFirestoreCommand.js` seem to be the culprit. After removing them locally though, the file path is correct but the JSON files at the fixture path are not being found with error `Error running firebase action: Fixture not found at path: `

**Comment by prescottprue** [comment]
@travwritescode Great to know! Makes total sense since that is from the new data passing feature - we should only do that if the argument is an Object (since when it is a string it is clearly a path).

Before we get the fix out you should be able to pass the loaded fixture as an object as the third argument.

**Comment by prescottprue** [comment]
Just to clarify, what I meant for loading a fixture before hand:

```js
cy.fixture('fakeProject.json').then((project) => {
  cy.callFirestore('add', 'project/test-project', project)
})
```

**Comment by prescottprue** [comment]
Fixed in v0.3.0. Feel free to reach out if things are working as expected, thanks for reporting!
